docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,112 @@
+---
+title: What maintainers should know before adopting Packit for downstream release automation 
+sidebar_position: 5
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages (Packit does not have yet a reverse dependencies check)
+
+:::tip `version_update_mask`
+
+The default Packit onboarding configuration enables automated releases only for `rawhide`. 
+
+When extending automation to branched Fedoras and EPELs, consider using [`version_update_mask`](https://packit.dev/docs/configuration#version_update_mask) to prevent breaking changes in stable Fedora versions.
+
+:::
+
+:::warning Packit pull requests can spam other maintainers
+
+SPam -> ask more about the spam
+
+:::
+
+:::warning missing notifications
+
+Packit bot user creates pull requests, Koji builds, and Bodhi updates, making it the default notification recipient.
+
+To receive pull request comment notifications, configure notification settings accordingly.
```

Comment:
How? Can we create/link a guide or at least the place to set it up? ([notifications.fedoraproject.org](https://notifications.fedoraproject.org/)?)